### PR TITLE
front: Test E2E composition code selector

### LINKED ISSUE #4142: test: Composition code selection
### Description and goal

- **Start page:** train creation page open

- **Steps:**
1. Click on the composition code selection component
2. Click on the search bar
3. Type "32"
4. Assert 1 result found 
5. Click on composition code "E32C"
  
- **Arrival page:** train creation page open

### Acceptance criteria

- [x] The composition code displayed is "Voyageur automoteur - E32C".

### PR DESCRIPTION
**Pull request by Uriel-Sautron**
Close #4142

### CODE REVIEW

**Review by SharglutDev — COMMENTED** [review]
Cool stuff and well organized :)

File: front/tests/assets/operationStudies/test_variables.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
 export default {
   numberOfRollingstock: process.env.CI ? '3' : '446',
-  numberOfRollingstockWithElectrical: process.env.CI ? '3' : '262',
+  numberOfRollingstockWithElectrical: process.env.CI ? '3' : '265',
```
Comment (clarani) [inline front/tests/assets/operationStudies/test_variables.ts:3]: Why do we have to increment this ? There are 3 new rolling stocks ? :open_mouth:

File: front/tests/add-train-page-model.ts
Comment (clarani) [inline front/tests/add-train-page-model.ts:1]: (Suggestion)
Can we rename this page ScenarioPage ?

**Review by clarani — COMMENTED** [review]
The test is broken because of the new UI

**Review by clarani — APPROVED** [review]

**Pull request merged**